### ISSUE TITLE
(Dashboard) - Opens under other drawers

### ISSUE THREAD

**Issue by caseychoiniere** [Duke-Translational-Bioinformatics/duke-data-service-portal#826]
If I'm in another drawer such as the "Add project members" drawer and then click in the left menu to navigate to the dashboard, it opens underneath the "Add project members" drawer (or the upload manager etc.)

Can probably fix this by making all drawers close if a user clicks outside of them. 

![screen shot 2018-03-07 at 2 08 16 pm](https://user-images.githubusercontent.com/10411174/37121269-7ae2f9d6-2211-11e8-9571-a5897b6147ba.png)

**Comment by caseychoiniere** [comment]
Actually I already have drawers set to close when a user clicks outside. 
`onRequestChange={() => this.toggleUploadManager()}`

I guess if a user clicks the menu icon or a menu item this doesn't get respected. When a user clicks to open the dashboard, might just need to check if other drawers are open and if so close them.

**Comment by caseychoiniere** [comment]
Could probably handle it here in dashboardStore but it might not be obvious whats happening

```javascript
 @action toggleDrawer() {
        this.drawer.set('open', !this.drawer.get('open'))
    }
```

**Comment by ChetCorey** [comment]
This is resolved in commit #730a2aec75335ffab6280a14d7aeba6e44523aed

    @action toggleUploadManager() {
        dashboardStore.drawer.set('open', this.openUploadManager);
        this.openUploadManager = !this.openUploadManager;
    }

![success dashboard - opens under other drawers 826](https://user-images.githubusercontent.com/7772027/39019407-317e1570-43f7-11e8-9b4a-ddeddb7ec17a.gif)
